### PR TITLE
Kimball Initial commit

### PR DESCRIPTION
**Pull request by mekimball**

### CODE REVIEW

File: lib/bus.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Bus
+  attr_reader :name, :capacity
+
+  def initialize(name)
```
Comment (timomitchel) [inline lib/bus.rb:4]: So this implementation should give you a wrong number of arguments error. We need to create a parameter named `capacity`  likes so: `def initialize(name, capacity)`

based on the way the test is set up.

File: lib/bus.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    @name = name
+    @capacity = []
```
Comment (timomitchel) [inline lib/bus.rb:6]: The array we needed to create was `@passengers = []`
capacity should be assigned to the parameter like so: `@capacity = capacity`

File: lib/bus.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  end
+
+  def capacity(passengers)
```
Comment (timomitchel) [inline lib/bus.rb:9]: This method should be called `add_passenger` and set up like so: 
```
def add_passenger(passenger)
@passengers << passenger
end
```

File: lib/bus.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    @capcity << passengers
+  end
+end
+
+# I apparently don't know this as well as I thought
```
Comment (timomitchel) [inline lib/bus.rb:14]: Don't worry. This is the first challenge. Challenges are a good way to find out what you need to spend more time working on.

**Review by timomitchel — COMMENTED** [review]
Hey Matthew, this is not a pass. I suggest you find time this week/weekend to go through this challenge with an upper mod student or a mentor.

File: lib/bus.rb (hunk 4 above)
Comment (mekimball) [reply lib/bus.rb:14]: I will definitely be going over it today. Already figured out some of the things I missed.

**Pull request closed without merging**